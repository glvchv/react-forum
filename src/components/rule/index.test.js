import React from 'react';
import renderer from 'react-test-renderer';
import Rule from '.';

describe('Rule component', () => {
    it('should render component', () => {
        const tree = renderer.create(<Rule/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});