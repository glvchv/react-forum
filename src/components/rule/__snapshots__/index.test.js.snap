// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rule component should render component 1`] = `
<div
  className="wrapper"
>
  <h1
    className="title"
  >
    . 
  </h1>
  <p
    className="text"
  />
</div>
`;
